<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarWidget</class>
 <widget class="QWidget" name="CarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>129</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>208</width>
    <height>129</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>208</width>
    <height>129</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CarWidget {
	background-color: qlineargradient(spread:pad, x1:1, y1:0.0227273, x2:1, y2:1, stop:0 rgba(39, 172, 255, 255), stop:0.961111 rgba(46, 102, 176, 255));
}
QPushButton {
	border:2px solid transparent;
}
QPushButton:hover {
	border:2px solid rgb(57, 136, 255);
}
#frameWidget {
	border: 2px solid black;
}
#frameWidget:hover {
	border: 2px solid rgb(32, 255, 177);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="frameWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>125</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="carMainWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>125</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>204</width>
          <height>125</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Auto:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="nameLabel">
                 <property name="styleSheet">
                  <string notr="true">font-weight: 600;
font-size: 14px;
color: rgb(148, 214, 255);</string>
                 </property>
                 <property name="text">
                  <string>Test</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignRight|Qt::AlignTop">
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="tosamo_2">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Tuning:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="tuningLabel">
                 <property name="styleSheet">
                  <string notr="true">font-weight: 600;
font-size: 14px;
color: rgb(148, 214, 255);</string>
                 </property>
                 <property name="text">
                  <string>36</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_787">
              <property name="styleSheet">
               <string notr="true">font-size: 11px;</string>
              </property>
              <property name="text">
               <string>    Cena:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="priceLabel">
              <property name="styleSheet">
               <string notr="true">font-size: 16px;
color: rgb(83, 190, 59);
font-weight: 600;</string>
              </property>
              <property name="text">
               <string>1000</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>punkt√≥w</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
